src/main/scala/is/hail/io/CassandraConnector.scala

```diff
@@ -108,23 +108,31 @@ object CassandraImpex {
     case TString => a
     case TBinary => ByteBuffer.wrap(a.asInstanceOf[Array[Byte]])
     case TCall => a
-    case TArray(elementType) =>
+    case TArray(elementType, req) =>
       if (a == null)
         null
       else
-        a.asInstanceOf[Seq[_]].map(x => exportAnnotation(x, elementType)).asJava
-    case TSet(elementType) =>
+        a.asInstanceOf[Seq[_]].map(x => {
+          assert(!req || x != null)
```

Comment:
I never against more asserts, but strictly speaking, we shouldn't need to check the values coming out of our representation (export), just those coming in.